test/test.py

```diff
@@ -146,8 +148,8 @@ def make_parser():
             self.assertEqual(set(self.run_completer(make_parser(), cmd)), set(output))
 
         expected_outputs = (
-            ("prog ", ["--foo", "--bar", "-h", "--help"]),
-            ("prog --b", ["--bar "])
+            ("prog ", ["--foo", "--bar", "--baz", "-h", "--help"]),
+            ("prog --b", ["--bar", "--baz"])
```

Comment:
I am not sure why I had to remove the trailing space for the existing option `--bar`?